spec/services/idv/profile_step_spec.rb

```diff
@@ -40,7 +40,7 @@
       expect(result.extra).to eq(extra)
       expect(idv_session.profile_confirmation).to eq true
       expect(idv_session.resolution_successful).to eq true
-      expect(idv_session.applicant).to eq(user_attrs.merge(uuid: user.uuid))
+      expect(idv_session.applicant).to eq(user_attrs.merge(uuid: user.uuid, uuid_prefix: nil))
```

Comment:
should we update one of these specs to have an SP with an `app_id` so that we can verify it's sent all the way?

Reply:
yes, meant to do that 🤦🏼 

added in https://github.com/18F/identity-idp/pull/4087/commits/2b37d583271767c3431106d0c993b94df24548b0